subworkflows/nf-core/vcf_annotate_ensemblvep_snpeff/main.nf

```diff
@@ -83,10 +86,12 @@ workflow VCF_ANNOTATE_ENSEMBLVEP_SNPEFF {
         ch_vep_input = ch_scatter.input
     } else {
         // Use the normal input when no scattering has to be performed
-        ch_vep_input = ch_vcf.map { meta, vcf, tbi, files -> [ meta, vcf, files ] }
+        ch_vep_input = ch_vcf.map { meta, vcf, _tbi, files -> [ meta, vcf, files ] }
```

Comment:
??

Reply:
Agreed, why the underscore? Is this  a new notation I don't know about?!